packages/gatsby/src/redux/actions.js

```diff
@@ -655,6 +656,24 @@ actions.createNode = (
   }
 }
 
+actions.createNode = (...args) => dispatch => {
+  const actions = createNode(...args)
+  dispatch(actions)
+  const createNodeAction = (Array.isArray(actions) ? actions : [actions]).find(
+    action => action.type === `CREATE_NODE`
+  )
+  if (createNodeAction) {
```

Comment:
this is just a suggestion, I find this a bit easier to read

```js
if (!createNodeAction) {
  return;
}

const { payload: node, traceId, parentSpan } = createNodeAction
return apiRunnerNode(`onCreateNode`, {
  node,
  traceId,
  parentSpan,
  traceTags: { nodeId: node.id, nodeType: node.internal.type },
})
```

Reply:
Thanks! Done.